src/modules/uORB/uORBDeviceMaster.hpp

```diff
@@ -69,6 +74,8 @@ class uORB::DeviceMaster
 	uORB::DeviceNode *getDeviceNode(const char *node_name);
 	uORB::DeviceNode *getDeviceNode(const struct orb_metadata *meta, const uint8_t instance);
 
+	bool deviceNodeExists(ORB_ID id, const uint8_t instance) { return _node_exists[instance][(uint8_t)id]; }
```

Comment:
Might be worth checking if `instance` is out of bounds.

Reply:
Done.